### PR TITLE
1. Added fix for label map json loader (as requested by Babel).\n 2. …

### PR DESCRIPTION
**Pull request by Nivi09**
…Added truthfulqa few shot static file.\n 3. Replaced os.system with subprocess call in user_postprocessor script executionmethod.

### CODE REVIEW

File: assets/aml-benchmark/components/inference-postprocessor/spec.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ type: command
 name: inference_postprocessor
 display_name: Inference Postprocessor
 description: Inference Postprocessor
-version: 0.0.1
+version: 0.0.1.1
```
Comment (arun-rajora) [inline assets/aml-benchmark/components/inference-postprocessor/spec.yaml:7]: Let's remove the component version update. When we release a component, we will update it then.

**Review by arun-rajora — CHANGES_REQUESTED** [review]

File: assets/aml-benchmark/components/inference-postprocessor/spec.yaml (hunk 1 above)
Comment (Nivi09) [reply assets/aml-benchmark/components/inference-postprocessor/spec.yaml:7]: okay. Done.

File: assets/aml-benchmark/tests/pipelines/inference_postprocessor_pipeline.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ type: pipeline
 display_name: Test Inference Postprocessor
 description: Pipeline to test Inference Postprocessor component
 settings:
-  default_compute: azureml:serverless
+  default_compute: azureml:NC24rsV3-dd         #serverless
```
Comment (arun-rajora) [inline assets/aml-benchmark/tests/pipelines/inference_postprocessor_pipeline.yaml:7]: Should it be serverless compute here?

Comment (Nivi09) [reply assets/aml-benchmark/tests/pipelines/inference_postprocessor_pipeline.yaml:7]: Sorry, yes, I will fix it.

Comment (Nivi09) [reply assets/aml-benchmark/tests/pipelines/inference_postprocessor_pipeline.yaml:7]: Done.

File: assets/aml-benchmark/scripts/data_loaders/truthfulqa_static_shots.py [hunk 3]
```diff
@@ -0,0 +1,153 @@
+# Copyright (c) Microsoft Corporation.
```
Comment (iamrk04) [inline assets/aml-benchmark/scripts/data_loaders/truthfulqa_static_shots.py:1]: @Nivi09, have we added any test for this custom script?

**Review by iamrk04 — APPROVED** [review]

**Review by arun-rajora — APPROVED** [review]

**Pull request merged**